packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.test.js

```diff
@@ -650,21 +650,20 @@ test('renders product bundle', () => {
     expect(screen.getByText(mockProductBundle.name)).toBeInTheDocument()
     expect(screen.getByRole('dialog', {name: /1 item added to cart/i})).toBeInTheDocument()
 
-    // bundle data is displayed in 1 row
     const numOfRowsRendered = screen.getAllByTestId('product-added').length
     expect(numOfRowsRendered).toBe(1)
 
-    // modal displays product name of children and variant selection
     mockBundleItemsAdded.forEach(({product, variant, quantity}) => {
-        const displayedName = quantity > 1 ? `${product.name} (${quantity})` : product.name
-        expect(screen.getByText(displayedName)).toBeInTheDocument()
-
+        expect(
+            screen.getByText((content) => content.trim().includes(product.name))
```

Comment:
Should the first argument is a Node instead of callback, the callback is the second arg. From this doc, you have to pass a container first. 🤔 https://testing-library.com/docs/dom-testing-library/cheatsheet/#text-match-options